IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -44,6 +46,14 @@ @interface MSIDCredentialCacheItem()
 
 @implementation MSIDCredentialCacheItem
 
+- (NSString *)description
+{
+    NSString *secretHash = [self.secret dataUsingEncoding:NSUTF8StringEncoding].msidSHA256.msidHexString;
+    if (secretHash.length > 8) secretHash = [secretHash substringToIndex:8];
```

Comment:
nit: provide a utility method for short token hash, as this might need to be used in multiple places

Reply:
done